src/database.c

```diff
@@ -12,6 +12,7 @@ and the Eclipse Distribution License is available at
  
 Contributors:
    Roger Light - initial implementation and documentation.
+   Tatsuzo Osawa - Fix subs memory issue.
```

Comment:
Please only add contributor lines for substantial contributions. This contribution changes just 17 lines and 18% of those lines are the contributor lines.